cmd/zetae2e/local/bitcoin.go

```diff
@@ -66,6 +66,7 @@ func bitcoinTestRoutine(
 		// to make it faster to catch up with the latest block header
 		if err := bitcoinRunner.RunE2ETestsFromNames(
 			e2etests.AllE2ETests,
+			e2etests.TestBitcoinDepositName,
```

Comment:
```suggestion
```
FYI: might not be explicit but it is not in this list because depositing Bitcoin is already done above to initialize the network for local testing.
We keep a `TestBitcoinDeposit` test to be used on live network where the network is already initialized.

Reply:
https://github.com/zeta-chain/node/pull/1791/commits/7c4bebe80d93b1474ca1913ddd24694e9de9c1d1